src/index.js

```diff
@@ -151,12 +149,14 @@ export const EDSCEchoform = ({
     }
   }
 
+  if (!model.firstElementChild) return null
```

Comment:
Comment on what this is looking for would be nice. If there is no firstElementChild what does that mean?